### PR TITLE
feat(projects): add description to add-projects and update-projects

### PR DESCRIPTION
**Pull request by lmjabreu**
## What

Surfaces project descriptions (26Q2 Project Essentials) in the MCP tools, matching the CLI surface.

- `add-projects` and `update-projects` accept an optional `description` parameter (Markdown; `update` accepts `""` to clear)
- The project output schema and `mapProject` now include `description`
- `find-projects`, `get-overview`, and `fetch` carry it through automatically (they reuse `mapProject` / the output schema)

## Design notes

- Keeps the SDK arg signature so only `description` escapes the types: `Parameters<typeof client.addProject>[0] & { description?: string }`. The SDK's `AddProjectArgs` / `UpdateProjectArgs` don't model `description` yet, but the REST client forwards it. Project **reads** already carry `description` in the SDK's `ProjectSchema`, so the output side needs no SDK change.
- Output `description` is a required string (the backend defaults it to `""`), consistent with the SDK read type.

## Scope

- **In scope:** project descriptions across add/update/find/overview/fetch.
- **Non-goals:** section descriptions (separate PR; the SDK strips `description` from section reads, so that one is SDK-gated).

## Testing

- New tests assert `description` is forwarded to the API and mapped back, in both `add-projects` and `update-projects`.
- Updated the schema-strictness key-list tests (description moves from disallowed to allowed) and the `mapProject` expectations.
- Full suite green (913 tests), type-check, lint/format, schema validation, and build all pass.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by lmjabreu** [comment]
@Doistbot /review

File: src/tools/add-projects.ts [hunk 1]
```diff
@@ -21,6 +21,10 @@ const ProjectSchema = z.object({
         .enum(['list', 'board', 'calendar'])
         .optional()
         .describe('The project view style. Defaults to "list".'),
+    description: z
```
Comment (doistbot) [inline src/tools/add-projects.ts:24]: <big><kbd>🟡 P2</kbd></big>  According to `AGENTS.md` (Documentation Requirements), adding a new tool feature requires updating the `src/mcp-server.ts` tool usage guidelines. Please update the documentation for `add-projects`/`update-projects` in `mcp-server.ts` to mention the newly supported `description` field and how to clear it.

File: src/tools/update-projects.ts [hunk 2]
```diff
@@ -11,6 +11,10 @@ const ProjectUpdateSchema = z.object({
     name: z.string().min(1).optional().describe('The new name of the project.'),
     isFavorite: z.boolean().optional().describe('Whether the project is a favorite.'),
     viewStyle: z.enum(['list', 'board', 'calendar']).optional().describe('The project view style.'),
+    description: z
+        .string()
+        .optional()
+        .describe('The description of the project. Supports Markdown. Pass "" to clear it.'),
```
Comment (doistbot) [inline src/tools/update-projects.ts]: <big><kbd>🟡 P2</kbd></big>  This clear contract is inconsistent with the repo’s established optional-field pattern. With the schema as written, `description: "remove"` would be saved literally as the project description, and legacy `null` would fail validation, unlike `update-goals`/`update-tasks`. Please expose a sentinel like `"remove"` here too, preprocess `null` for back-compat, and translate the sentinel to the API’s clear value before calling `updateProject()`.

File: src/tool-helpers.ts [hunk 3]
```diff
@@ -274,6 +274,7 @@ function mapProject(project: Project) {
     return {
         id: project.id,
         name: project.name,
+        description: project.description,
```
Comment (doistbot) [inline src/tool-helpers.ts:277]: <big><kbd>🟡 P2</kbd></big>  Adding `description` here only covers tools that actually call `mapProject`. `get-overview` builds its own project payloads (`ProjectStructureSchema`, `projectInfo`, and `project`) and still omits this field, so overview responses won’t surface descriptions despite the PR scope. Please thread `description` through `get-overview`’s schemas/builders as well, or narrow the feature claim.

File: src/tools/__tests__/update-projects.test.ts [hunk 4]
```diff
@@ -78,6 +78,28 @@ describe(`${UPDATE_PROJECTS} tool`, () => {
             )
         })
 
+        it('should forward description to the API and map it back', async () => {
```
Comment (doistbot) [inline src/tools/__tests__/update-projects.test.ts:81]: <big><kbd>🟡 P2</kbd></big>  `AGENTS.md` testing requirements state: "Test clearing values (if applicable)". Since `description` in updates explicitly supports clearing via an empty string (`""`), please add a test case verifying this clear behavior.

File: src/tools/update-projects.ts [hunk 5]
```diff
@@ -11,6 +11,10 @@ const ProjectUpdateSchema = z.object({
     name: z.string().min(1).optional().describe('The new name of the project.'),
     isFavorite: z.boolean().optional().describe('Whether the project is a favorite.'),
     viewStyle: z.enum(['list', 'board', 'calendar']).optional().describe('The project view style.'),
+    description: z
```
Comment (doistbot) [inline src/tools/update-projects.ts:14]: <big><kbd>🟡 P2</kbd></big>  Adding `description` here makes the update path stringly-typed. `updateProjects.execute` still forwards the inferred `updateArgs` object straight to `client.updateProject`, so this new key is no longer checked against the SDK signature the way `add-projects` now does. Please build a typed payload such as `Parameters<typeof client.updateProject>[1] & { description?: string }` before the SDK call so future SDK changes or field typos stay compile-checked.

**Review by doistbot — COMMENTED** [review]
This PR nicely introduces the `description` field to our project tools and models while cleanly routing around the SDK's current type constraints.

Few things worth tightening:
- **Clear contract & typing:** Update `update-projects` to use our standard `"remove"` sentinel (handling legacy `null`) instead of `""` for clearing descriptions, and explicitly type the payload to ensure the SDK boundary remains checked.
- **Overview scope:** Thread the new field through `get-overview`'s bespoke project schemas so descriptions actually surface there as intended.
- **Docs & test coverage:** Add a test case for the clear behavior and update the tool usage guidelines in `src/mcp-server.ts` to fulfill the `AGENTS.md` checklist.

<sub>[Share Feedback](https://github.com/Doist/doistbot-app/issues/new?template=review-feedback.yml&title=Review+feedback+for+Doist%2Ftodoist-mcp&pr_url=https%3A%2F%2Fgithub.com%2FDoist%2Ftodoist-mcp%2Fpull%2F500&datadog_url=https%3A%2F%2Fapp.datadoghq.com%2Flogs%3Fquery%3Dservice%253A%2528doistbot-app-sandbox%2520OR%2520doistbot-app%2529%2520%2540meta.deliveryId%253Ae00a8ac0-6512-11f1-9b7e-485c58165cf8%26agg_m%3Dcount%26agg_m_source%3Dbase%26agg_t%3Dcount%26clustering_pattern_field_path%3Dmessage%26cols%3Dhost%252Cservice%26fromUser%3Dtrue%26messageDisplay%3Dinline%26refresh_mode%3Dsliding%26storage%3Dhot%26stream_sort%3Ddesc%26viz%3Dstream%26live%3Dtrue) • [Review Logs](https://app.datadoghq.com/logs?query=service%3A%28doistbot-app-sandbox%20OR%20doistbot-app%29%20%40meta.deliveryId%3Ae00a8ac0-6512-11f1-9b7e-485c58165cf8&agg_m=count&agg_m_source=base&agg_t=count&clustering_pattern_field_path=message&cols=host%2Cservice&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&live=true)</sub>

File: src/tools/update-projects.ts (hunk 2 above)
Comment (lmjabreu) [reply src/tools/update-projects.ts]: Done. Project description clearing now uses the `"remove"` sentinel (and legacy `null`), mapped to `""` (the project clear value, backend NULL_KEEPS_UNCHANGED) — matching update-goals/update-tasks.

File: src/tools/update-projects.ts (hunk 5 above)
Comment (lmjabreu) [reply src/tools/update-projects.ts:14]: Done. The update payload is now explicitly typed as `Parameters<typeof client.updateProject>[1] & { description?: string }`, so only `description` escapes the SDK types.

File: src/tools/add-projects.ts (hunk 1 above)
Comment (lmjabreu) [reply src/tools/add-projects.ts:24]: Done. Documented descriptions (incl. the "remove" clear) in the project tool guidance in mcp-server.ts.

File: src/tools/__tests__/update-projects.test.ts (hunk 4 above)
Comment (lmjabreu) [reply src/tools/__tests__/update-projects.test.ts:81]: Done. Added tests for clearing via `"remove"` and via legacy `null` (through the schema preprocess).

File: src/tool-helpers.ts (hunk 3 above)
Comment (lmjabreu) [reply src/tool-helpers.ts:277]: Keeping as-is. get-overview's project shapes are intentionally minimal — the tree omits color, isFavorite, isShared and most fields, carrying only structural data (id, name, parent/folder, order, sections). A potentially long markdown description doesn't fit that compact view; it's available via find-projects and the add/update project tools. Happy to add it if you'd prefer consistency over minimalism.

**Comment by lmjabreu** [comment]
@Doistbot /review

File: src/tool-helpers.ts (hunk 3 above)
Comment (doistbot) [inline src/tool-helpers.ts:277]: <big><kbd>🟡 P2</kbd></big>  Adding `description` to `mapProject()` isn't enough to make `fetch` return it. `src/tools/fetch.ts` builds its own `FetchResult` and only includes the project name in `text` plus a few metadata fields, so `project:{id}` fetch responses still omit the description despite the PR contract. Please thread `mappedProject.description` into that payload as well, or narrow the feature claim/documentation.

**Review by doistbot — COMMENTED** [review]
This PR introduces project description support across our project-related tools and thoughtfully aligns with the established optional-field clearing patterns.

Few things worth tightening:
- Thread `mappedProject.description` into the `fetch` tool's payload so descriptions are actually exposed during fetch reads, or narrow the feature claim in the documentation.

I also included a few optional follow-up notes in the details below.

<details>
<summary>Optional follow-up note (1)</summary>

- **[P3] src/tool-helpers.test.ts:182**: These new workspace `mapProject()` assertions only exercise the default empty description. Since the add/update tests cover personal projects, we still don't have a regression guard for non-empty descriptions on workspace projects—the read path this PR is surfacing in `find-projects`/`fetch`. Set one workspace fixture's `description` to a real value and assert it survives mapping.

</details>

<sub>[Share Feedback](https://github.com/Doist/doistbot-app/issues/new?template=review-feedback.yml&title=Review+feedback+for+Doist%2Ftodoist-mcp&pr_url=https%3A%2F%2Fgithub.com%2FDoist%2Ftodoist-mcp%2Fpull%2F500&datadog_url=https%3A%2F%2Fapp.datadoghq.com%2Flogs%3Fquery%3Dservice%253A%2528doistbot-app-sandbox%2520OR%2520doistbot-app%2529%2520%2540meta.deliveryId%253Adb5cf980-6518-11f1-84bc-ce7d2bb4d3cb%26agg_m%3Dcount%26agg_m_source%3Dbase%26agg_t%3Dcount%26clustering_pattern_field_path%3Dmessage%26cols%3Dhost%252Cservice%26fromUser%3Dtrue%26messageDisplay%3Dinline%26refresh_mode%3Dsliding%26storage%3Dhot%26stream_sort%3Ddesc%26viz%3Dstream%26live%3Dtrue) • [Review Logs](https://app.datadoghq.com/logs?query=service%3A%28doistbot-app-sandbox%20OR%20doistbot-app%29%20%40meta.deliveryId%3Adb5cf980-6518-11f1-84bc-ce7d2bb4d3cb&agg_m=count&agg_m_source=base&agg_t=count&clustering_pattern_field_path=message&cols=host%2Cservice&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&live=true)</sub>

File: src/tool-helpers.ts (hunk 3 above)
Comment (doistbot) [inline src/tool-helpers.ts:277]: <big><kbd>🟡 P2</kbd></big>  The PR description claims that `fetch` automatically carries the description through since it reuses `mapProject`. However, `src/tools/fetch.ts` manually destructs `mappedProject` into a custom `result` object (containing only `text`, `url`, and `metadata`) and completely drops this new field. Please thread `mappedProject.description` into the `fetch` tool payload (e.g., appending it to the `text` output) so descriptions actually surface during fetch reads, or narrow the feature claim in the documentation.

**Review by doistbot — COMMENTED** [review]
This PR surfaces project descriptions across the MCP project tools to mirror the workspace capabilities.

Few things worth tightening:
- Check the Internal AI Tools standard to ensure exposing raw project descriptions to the LLM doesn't require a specific approval or filtering path.
- Thread the new description field into the `fetch` tool's custom payload so it actually surfaces during reads as intended.
- Build an explicitly typed API payload in `add-projects` instead of spreading the whole tool payload directly into the SDK.
- Reject empty strings in the schema to ensure `"remove"` remains the single, documented convention for clearing fields.

I also included a few optional follow-up notes in the details below.

<details>
<summary>Optional follow-up note (1)</summary>

- **[P3] src/tool-helpers.test.ts:182**: These new `mapProject()` assertions only exercise the default empty description. Since the add/update tests already cover the write path on personal projects, we still don't have a regression test that a non-empty description survives mapping for workspace projects—the read path behind `find-projects`/`fetch`. Set one `createMockWorkspaceProject()` fixture's `description` to a real value and assert it comes through.

</details>

<sub>[Share Feedback](https://github.com/Doist/doistbot-app/issues/new?template=review-feedback.yml&title=Review+feedback+for+Doist%2Ftodoist-mcp&pr_url=https%3A%2F%2Fgithub.com%2FDoist%2Ftodoist-mcp%2Fpull%2F500&datadog_url=https%3A%2F%2Fapp.datadoghq.com%2Flogs%3Fquery%3Dservice%253A%2528doistbot-app-sandbox%2520OR%2520doistbot-app%2529%2520%2540meta.deliveryId%253Accb9e040-6587-11f1-8d48-823a6f08b961%26agg_m%3Dcount%26agg_m_source%3Dbase%26agg_t%3Dcount%26clustering_pattern_field_path%3Dmessage%26cols%3Dhost%252Cservice%26fromUser%3Dtrue%26messageDisplay%3Dinline%26refresh_mode%3Dsliding%26storage%3Dhot%26stream_sort%3Ddesc%26viz%3Dstream%26live%3Dtrue) • [Review Logs](https://app.datadoghq.com/logs?query=service%3A%28doistbot-app-sandbox%20OR%20doistbot-app%29%20%40meta.deliveryId%3Accb9e040-6587-11f1-8d48-823a6f08b961&agg_m=count&agg_m_source=base&agg_t=count&clustering_pattern_field_path=message&cols=host%2Cservice&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&live=true)</sub>

File: src/tools/add-projects.ts (hunk 1 above)
Comment (doistbot) [inline src/tools/add-projects.ts:24]: <big><kbd>🟡 P2</kbd></big>  Adding `description` here widens the tool schema, but `execute()` still spreads the whole tool payload straight into `client.addProject()`. That means any tool-only key added here bypasses the SDK boundary check and gets forwarded on the wire. Please build a typed API payload first (for example `Parameters<typeof client.addProject>[0] & { description?: string }`) and pass that to `addProject()`.

File: src/tools/update-projects.ts [hunk 6]
```diff
@@ -6,11 +6,23 @@ import { ColorSchema } from '../utils/colors.js'
...
 const ProjectUpdateSchema = z.object({
     id: z.string().min(1).describe('The ID of the project to update.'),
     name: z.string().min(1).optional().describe('The new name of the project.'),
     isFavorite: z.boolean().optional().describe('Whether the project is a favorite.'),
     viewStyle: z.enum(['list', 'board', 'calendar']).optional().describe('The project view style.'),
+    description: z
+        .preprocess(
+            (value) => (value === null ? REMOVE_SENTINEL : value),
+            z
+                .string()
```
Comment (doistbot) [inline src/tools/update-projects.ts:21]: <big><kbd>🟡 P2</kbd></big>  This schema still accepts `description: ""`, and the new mapping logic forwards that as the backend's clear value. That leaves two clear conventions (`"remove"` and `""`) even though the repo standard is to use an explicit sentinel for clears. Reject empty strings here (for example with `.min(1)` or a refine) so `"remove"` stays the only documented clear path, aside from legacy `null` preprocessing.

File: src/tool-helpers.ts (hunk 3 above)
Comment (doistbot) [inline src/tool-helpers.ts:277]: <big><kbd>🟡 P2</kbd></big>  This change starts returning raw project descriptions from `mapProject()`, which means user-generated project content now flows into the MCP response surface for LLM clients. Doist's Internal AI Tools standard says builders should actively avoid feeding task/project content to model context unless there's an approved exception and filtering path. Please keep `description` out of the LLM-facing project shape, or document and implement the required approval/filtering path before exposing it. https://handbook.doist.com/doc/standard-internal-ai-tools-zU0fqnhpmC

**Comment by gnapse** [comment]
Aren't these a feature that's behind a feature flag? We cannot just add it here if that's so. Otherwise, LLMs will add project descriptions that people will then not see in their app. Or am I missing something?

File: src/tools/update-projects.ts (hunk 6 above)
Comment (lmjabreu) [reply src/tools/update-projects.ts:21]: Fixed. `.min(1)` rejects `""`, so `"remove"` is the single documented clear path (legacy `null` is preprocessed to it), matching update-goals/update-tasks. Rejection test added in bcc0673.

File: src/tool-helpers.ts (hunk 3 above)
Comment (lmjabreu) [reply src/tool-helpers.ts:277]: Fixed. `fetch` now threads `mappedProject.description` into its `text`, mirroring the task path. Test added in bcc0673.

File: src/tools/update-projects.ts (hunk 2 above)
Comment (lmjabreu) [reply src/tools/update-projects.ts]: Done. The schema implements the `"remove"` sentinel contract exactly now (legacy `null` preprocessed, `""` rejected). In bcc0673.

File: src/tool-helpers.ts (hunk 3 above)
Comment (lmjabreu) [reply src/tool-helpers.ts:277]: Fixed. In bcc0673: `fetch` includes the description now (the PR body claim predated this); test added.

File: src/tools/update-projects.ts (hunk 5 above)
Comment (lmjabreu) [reply src/tools/update-projects.ts:14]: Done. The casts went with the 10.4.0 bump: `update-projects` forwards a value typed from the SDK's parameter type.

File: src/tools/add-projects.ts (hunk 1 above)
Comment (lmjabreu) [reply src/tools/add-projects.ts:24]: Done. `add-projects` spreads into the SDK-typed `addProject` call since the 10.4.0 bump, so tool-only keys fail the compile.

Comment (lmjabreu) [reply src/tools/add-projects.ts:24]: Done. The `mcp-server.ts` usage guidelines cover the description support; on the current tip.

File: src/tools/__tests__/update-projects.test.ts (hunk 4 above)
Comment (lmjabreu) [reply src/tools/__tests__/update-projects.test.ts:81]: Done. Clear-path tests are on the current tip (`"remove"` and legacy `null`), plus the new `""` rejection test in bcc0673.

File: src/tool-helpers.ts (hunk 3 above)
Comment (lmjabreu) [reply src/tool-helpers.ts:277]: Keeping as-is. `get-overview`'s project shapes are deliberately minimal (they omit color, favorite, and shared too); descriptions are available via find-projects, add/update, and fetch.

Comment (lmjabreu) [reply src/tool-helpers.ts:277]: Keeping as-is. The MCP already returns user-generated content to clients by design: task descriptions, comment content, and project/section/label names. Project descriptions are the same data class, and filtering only them would be inconsistent while gutting the feature. If the Internal AI Tools standard requires sanitization or wrapping, that's an MCP-wide concern (tasks, comments, names) and worth a separate issue rather than scoping it to this PR.

**Comment by chiara-doist** [comment]
@gnapse I agree, the MCP and CLI should be paused until the full feature release (🔜). Thanks for thinking about it and flagging it!

**Comment by lmjabreu** [comment]
@gnapse We're discussing this internally in the squad ([thread](https://twist.com/a/1585/ch/820137/t/7826892/c/102412506)). For precedent: Goals was included here while it was an even more experimental feature. Descriptions aren't an if, they're a when: the backend is live, the web surfaces are flag-gated, and the open question is sequencing. We'll report back here before moving the MCP PRs.

**Comment by gnapse** [comment]
> Goals was included here while it was an even more experimental feature.

Yeah, good point. I participated on a few PR reviews about it in the CLI and MCP without having realized it was not yet official. I assumed it was enabled for everyone, given that we were delivering it in the CLI and MCP servers. We shouldn't have done that. And I would've flagged it if I knew.

File: src/mcp-server.ts [hunk 7]
```diff
@@ -86,7 +86,7 @@ You have access to comprehensive Todoist management tools for personal productiv
 - **find-completed-tasks**: View completed tasks by completion date or original due date; if since/until are omitted, defaults to the last 7 days (returns all collaborators unless filtered)
 
 **Project & Organization:**
-- **add-projects/update-projects/find-projects**: Manage project lifecycle with names, favorites, view styles (list/board/calendar), and workspace assignment for new projects (by name or ID)
+- **add-projects/update-projects/find-projects**: Manage project lifecycle with names, descriptions (Markdown; on update pass "remove" to clear), favorites, view styles (list/board/calendar), and workspace assignment for new projects (by name or ID)
```
Comment (gnapse) [inline src/mcp-server.ts]: What if the description of a task is "remove"? Plus, this is not the place to document how to make the removal. That should go into the tool description or that particular argument's description.

File: src/tools/__tests__/update-projects.test.ts [hunk 8]
```diff
@@ -78,6 +78,66 @@ describe(`${UPDATE_PROJECTS} tool`, () => {
...
+
+        it('clears the description via the "remove" sentinel (empty string on the wire)', async () => {
+            mockTodoistApi.updateProject.mockResolvedValue(
+                createMockProject({ id: 'project-123', name: 'Docs', description: '' }),
+            )
+
+            await updateProjects.execute(
+                { projects: [{ id: 'project-123', description: 'remove' }] },
+                mockTodoistApi,
+            )
```
Comment (gnapse) [inline src/tools/__tests__/update-projects.test.ts:111]: As mentioned above, passing the string `"remove"` is not a good way to offer the ability to remove the description. What if someone actually wants a task description to be that exact string? (sounds unlikely, but that is irrelevant; doing that in the app and via the API *is* possible, so it should be possible while using the MCP server as well).

Why can't this be simply `null`?

```suggestion
            await updateProjects.execute(
                { projects: [{ id: 'project-123', description: null }] },
                mockTodoistApi,
            )
```

File: src/tools/__tests__/update-projects.test.ts [hunk 9]
```diff
@@ -78,6 +78,66 @@ describe(`${UPDATE_PROJECTS} tool`, () => {
...
+                description: '',
+            })
+        })
+
+        it('rejects an empty-string description ("remove" is the only clear path)', () => {
+            const result = z.object(updateProjects.parameters).safeParse({
+                projects: [{ id: 'project-123', description: '' }],
+            })
+            expect(result.success).toBe(false)
+        })
```
Comment (gnapse) [inline src/tools/__tests__/update-projects.test.ts]: I also do not understand this design choice. Should be possible to set the description to any valid string, including "remove" and the empty string as well.

**Review by gnapse — DISMISSED** [review]

File: src/tools/__tests__/update-projects.test.ts (hunk 8 above)
Comment (lmjabreu) [reply src/tools/__tests__/update-projects.test.ts:111]: > As mentioned above, passing the string `"remove"` is not a good way to offer the ability to remove the description. What if someone actually wants a task description to be that exact string? [...] Why can't this be simply `null`?

I copied the sentinel pattern from update-goals, where it allowed Gemini to clear the description. I thought it was a conscious trade-off, but perhaps not. And yes, the collision of someone setting the description to "remove" and the remove sentinel would be weird for users. I've removed it now. A plain empty string or null clears it. Thank you.

File: src/mcp-server.ts (hunk 7 above)
Comment (lmjabreu) [reply src/mcp-server.ts]: Fixed. Removed the instructions from here; the clearing guidance lives in the description argument's own description now.

File: src/tools/__tests__/update-projects.test.ts (hunk 9 above)
Comment (lmjabreu) [reply src/tools/__tests__/update-projects.test.ts]: Done. The decision to drop "remove" is in the thread above.

**Comment by chiara-doist** [comment]
(I _think_ Goals was released under a pre-release version)

**Comment by lmjabreu** [comment]
> (I _think_ Goals was released under a pre-release version)

True, the SDK was a pre-release (-next channel), the MCP had no such gating. Anyway, we can proceed as per Dom's approval :) Thank you!

**Comment by lmjabreu** [comment]
@gnapse Would you be able to review this when you have a moment? Descriptions is already on web and mobile internally 😊 No rush as we can release without this.

**Review by scottlovegrove — APPROVED** [review]
